logging/google/cloud/logging/logger.py

```diff
@@ -209,16 +215,20 @@ def log_text(self, text, client=None, labels=None, insert_id=None,
 
         :type timestamp: :class:`datetime.datetime`
         :param timestamp: (optional) timestamp of event being logged.
```

Comment:
Hmm, in this case `timestamp` needs to move up, although that issue was present before the PR.